gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/publisher/GobblinMCEPublisher.java

```diff
@@ -198,7 +198,12 @@ public static Metrics getMetrics(State state, Path path, Configuration conf, Nam
           //This means the table is not compatible with iceberg, so return a dummy metric
           return new Metrics(100000000L, null, null, null);
         }
-        return OrcMetrics.fromInputFile(HadoopInputFile.fromPath(path, conf), MetricsConfig.getDefault(), mapping);
+        try {
+          return OrcMetrics.fromInputFile(HadoopInputFile.fromPath(path, conf), MetricsConfig.getDefault(), mapping);
+        } catch (Exception e) {
+          //This means the table is not compatible with iceberg, so return a dummy metric
+          return new Metrics(100000000L, null, null, null);
```

Comment:
Can we have a common static instance called DUMMY_METRIC = new Metrics(100000000L, null, null, null); and return in both places

Reply:
addressed your comment